model-optimizer/requirements_mxnet.txt

```diff
@@ -1,6 +1,7 @@
-mxnet>=1.0.0,<=1.7.0
-networkx>=1.11
-numpy>=1.14.0
-defusedxml>=0.5.0
-urllib3>=1.25.9
-requests>=2.20.0
+mxnet~=1.2.0 ; sys_platform == 'win32'
```

Comment:
Are whitespaces between module version and `;` necessary? If not, please remove them. Note as we use environment markers `tensorflow>=1.15.2,<2.0; python_version < "3.8"`